components/DashboardPage/components/index.ts

```diff
@@ -1,5 +1,5 @@
 export { Sidebar } from './Sidebar';
 export { WeekMonthYearToggle } from './WeekMonthYearToggle';
 export { WidgetsArea } from './WidgetsArea';
-export { WeeklyTodos } from './WeeklyTodos';
+export { WeeklyTodos } from './WeeklyTodos/WeeklyTodos';
```

Comment:
If you want to avoid all the duplicated names in the folder imports, I suggest adding an `index.ts` file at the component root level in each component.